cypress/e2e/swap.test.ts

```diff
@@ -112,6 +115,53 @@ describe('Swap', () => {
       })
   })
 
+  it('should render an error when a transaction fails due to a passed deadline', () => {
+    const BALANCE_INCREMENT = 1
+    const DEADLINE_MINUTES = 30
+    cy.visit('/swap', { ethereum: 'hardhat' })
+      .then((window) => {
+        hardhat = window.hardhat
+      })
+      .then(() => hardhat.utils.getBalance(hardhat.wallet.address, USDC_MAINNET))
+      .then((balance) => Number(balance.toFixed(1)))
+      .then((initialBalance) => {
+        // input swap info
+        cy.get('#swap-currency-output .open-currency-select-button').click()
+        cy.get(getTestSelector('token-search-input')).clear().type(MAINNET_USDC_ADDRESS)
+        cy.contains('USDC').click()
+        cy.get('#swap-currency-output .token-amount-input').clear().type(BALANCE_INCREMENT.toString())
+        cy.get('#swap-currency-input .token-amount-input').should('not.equal', '')
```

Comment:
Why not assert what it _will_ equal?

Reply:
That requires complicated pre-calc of routing stuff given chain state at the fork point